background/get-addon-settings.js

```diff
@@ -170,6 +172,21 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
       addonSettings["workspace-dots"]["theme"] = "none";
     }
 
+    if (addonSettings["editor-theme3333"]?._version === 3) {
```

Comment:
```suggestion
    if (addonSettings["editor-theme3"]?._version === 3) {
```

Reply:
Does this really need to interact with theme3? I think it would make more sense to add a property to msg-count-badge settings indicating that the migration has been applied.

Reply:
Yes, that would also work.

Reply:
If we are discussing this I guess it means we all somewhat agree that this change (not how it's implemented) isn't too bad

Reply:
I agree with the change.

Reply:
> Does this really need to interact with theme3?

It only reads from editor-theme3, I don't think it's too bad.
The only way the version doesn't get increased to 4 is a crash, so this can't run more than twice.